saleor/graphql/product/types/products.py

```diff
@@ -276,7 +276,12 @@ class ProductVariant(ChannelContextTypeWithMetadata[models.ProductVariant]):
         description="The product to which the variant belongs.",
     )
     track_inventory = graphene.Boolean(
-        required=True, description="Whether the inventory of the variant is tracked."
+        required=True,
+        description=(
+            "Determines if the inventory of this variant should be tracked. If false, "
+            "the quantity won't change when customers buy this item."
```

Comment:
Missing space. It should be `but this item. "` (see the description generated to `schema.graphql` the space there is missing.

Reply:
@korycins Please review.